dish_grpc_mqtt.py

```diff
@@ -72,6 +83,29 @@ def wrap_ca_arg(arg):
     else:
         parser.epilog += "\nSSL support options not available due to missing ssl module"
 
+    env_map = (
+        ("MQTT_HOST", "hostname"),
+        ("MQTT_PORT", "port"),
+        ("MQTT_USERNAME", "username"),
+        ("MQTT_PASSWORD", "password"),
+        ("MQTT_SSL", "tls"),
+    )
+    env_defaults = {}
+    for var, opt in env_map:
+        # check both set and not empty string
+        val = os.environ.get(var)
+        if val:
+            if var == "MQTT_SSL":
```

Comment:
The logic under this conditional does not look correct for the paho `tls` kwarg.

I can fix this post-submit if you prefer, as I think I'm still set up to test SSL MQTT broker.

Reply:
I don't have SSL MQTT set up so if you do, it might be easiest for you to fix.  Same with the systemd service file below.